ui-tests/src/test/java/org/openmrs/reference/UsingBackButtonInMergePatientTest.java

```diff
@@ -76,15 +73,12 @@ public void usingBackButtonInMergePatientTest() throws Exception {
         assertNotNull(dataManagementPage.CONTINUE);
     }
 
-
     @After
     public void tearDown() throws Exception {
         headerPage.clickOnHomeIcon();
         TestData.PatientInfo p = new TestData.PatientInfo();
         p.uuid = patient.uuid;
         deletePatient(p);
         waitForPatientDeletion(patient.uuid);
-        headerPage.logOut();
```

Comment:
There's no need to invoke the headerPage.logOut() in the tearDown() function after the test patient has been deleted from the database. The test still passes without this function.